tests/countries/test_singapore.py

```diff
@@ -9,15 +9,19 @@
 #  Website: https://github.com/dr-prodigy/python-holidays
 #  License: MIT (see LICENSE file)
 
-import unittest
 from datetime import date
 
-import holidays
+from holidays.countries.singapore import Singapore, SG, SGP
+from tests.common import TestCase
 
 
-class TestSingapore(unittest.TestCase):
-    def setUp(self):
-        self.holidays = holidays.Singapore()
+class TestSingapore(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Singapore)
+
+    def test_country_aliases(self):
```

Comment:
We need to add something for test coverage of `__init__()` of SG and SGP classes. For Malaysia, I added
```
        holidays1 = MY()
        holidays2 = MYS()
        self.assertEqual(list(holidays1), list(holidays2))
```